src/mapInteractor.js

```diff
@@ -192,11 +203,35 @@ var mapInteractor = function (args) {
          * (which would propably require detecting keyup events). */
         focusHighlight: true
       },
-
+      /* Set alwaysTouch to false to only add touch support on devices that
+       * report touch support.  Set to true to add touch support on all
+       * devices. */
+      alwaysTouch: false,
       wheelScaleX: 1,
       wheelScaleY: 1,
       zoomScale: 1,
       rotateWheelScale: 6 * Math.PI / 180,
+      /* The minimum angle of rotation (in radians) before the
+       * geo_action.zoomrotate action will allow rotation.  Set to 0 to always
+       * include rotation. */
+      zoomrotateMinimumRotation: 5.0 * Math.PI / 180,
+      /* The minimum angle of rotation (in radians) before the
+       * geo_action.zoomrotate action will reverse the rotation direction.
+       * This helps reduce chatter when zooms and pans are combined with
+       * rotations. */
+      zoomrotateReverseRotation: 4.0 * Math.PI / 180,
```

Comment:
The first value (5) is for initial rotation to prevent accidental rotation when you are trying to just zoom or just pan.  The reverse value (4) is when you are rotating to debounce rotating back and forth -- it should be less than or equal to the initial rotation threshold.  I tried a smaller number, but sometimes when panning and rotating, the map would jitter in rotation (because in a two-finger pan, the two touch points don't necessarily move in perfect synch).  I settled on 4 as working.  Feel free to play with these values and suggest other ones.